icq-relayer/pkg/runner/run.go

```diff
@@ -738,15 +738,15 @@ func prepareMessages(msgSlice []Message, logger log.Logger) []sdk.Msg {
 		}
 
 		if _, ok := keys[msg.QueryId]; ok {
-			//fmt.Println("message already added")
+			// fmt.Println("message already added")
 			continue // message already added
 		}
 
 		if entry.ClientUpdate != nil && !exists(keys, fmt.Sprintf("%s-%d", entry.ClientUpdate.ConnectionId, entry.ClientUpdate.Height)) {
-			//fmt.Println("client update required")
+			// fmt.Println("client update required")
 			cu, err := getCachedClientUpdate(entry.ClientUpdate.ConnectionId, entry.ClientUpdate.Height)
 			if err != nil {
-				//fmt.Println("client update not ready; requeueing")
+				// fmt.Println("client update not ready; requeueing")
 				go func(entry Message) { time.Sleep(time.Second * 2); sendQueue <- entry }(entry) // client update not ready; requeue.
 			} else {
```

Comment:
should we move commented  print statements to some logs for debugging if needed?